src/components/StyledMain.js

```diff
@@ -5,7 +5,7 @@ const StyledMain = styled.main`
 	flex-direction: column;
 	align-items: center;
 	gap: 20px;
-	height: calc(100vh - 30px);
+	margin: 0 0 30px;
```

Comment:
This throws no error? 👀
I thought you could either enter parameter for all 4 sides OR  2 (above/below; left/right)